webpack.config.js

```diff
@@ -1,63 +1,33 @@
-// Generated using webpack-cli https://github.com/webpack/webpack-cli
-
-const path = require('path');
-const HtmlWebpackPlugin = require('html-webpack-plugin');
-const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-
-const isProduction = process.env.NODE_ENV == 'production';
-
-
-const stylesHandler = isProduction ? MiniCssExtractPlugin.loader : 'style-loader';
-
-
-
-const config = {
-    entry: './src/index.js',
-    output: {
-        path: path.resolve(__dirname, 'dist'),
-    },
-    devServer: {
-        open: true,
-        host: 'localhost',
-    },
-    plugins: [
-        new HtmlWebpackPlugin({
-            template: 'index.html',
-        }),
-
-        // Add your plugins here
-        // Learn more about plugins from https://webpack.js.org/configuration/plugins/
+require("dotenv").config();
+
+const path = require("path");
+
+module.exports = {
+  mode: "development",
+  entry: path.join(__dirname, "/client/src/index.jsx"),
+  output: {
+    path: path.join(__dirname, "/client/dist"),
+    filename: "bundle.js",
+  },
+  devtool: "source-map",
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /nodeModules/,
```

Comment:
this should be `node_modules` for the correct path
```suggestion
        exclude: /node_modules/,
```